ios-shoppe-demo/ios-shoppe-demo/ios-shoppe-demo/APIService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  APIService.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/4/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+
+class APIService {
+    static var sharedInstance = APIService()
+
+    var baseURL: String = "http://reactshoppe.s3-website-us-east-1.amazonaws.com/data/products.json"
+    var shopItems: [[String : Any]] = [] // TODO: Change this to be an array of class ShopItem
```

Comment:
Would it be a good idea to include this in a config file rather than hard-code it here?

Reply:
I Added a config file (Preferences) to hold hardcoded Values
Made baseURL a computed property that retrieves the URL from the config file.